projects.yaml

```diff
@@ -554,6 +554,13 @@ projects:
   category: datasets
   description: Database of yet-to-be-sythesized materials predicted using state-of-the-art machine learning algorithms
   homepage: https://matterverse.ai/
+- name: Open Databases Integration for Materials Design (OPTIMADE)
+  resource: true
```

Comment:
Will remove `rescource: true`. Why: Resources are not ranked. Whenever a project has an associated repo (this is considered good), it should be ranked. This is the case here.